.github/workflows/publish-release-android-bte.yml

```diff
@@ -0,0 +1,117 @@
+name: Publish Release Candidates
+
+on:
+  push:
+    branches:
+      - bte
+      - release-candidate
```

Comment:
not likely to be an issue, but it might make sense to remove `- release-candidate` and `- master` until we have a plan on how we handle bte / gps deploys. there are some open questions here, like should they always be released together or separate?, etc.

Reply:
👍